app/views/public/api/v1/portals/_featured_articles.html.erb

```diff
@@ -0,0 +1,28 @@
+<% featured_articles = articles.where(status: :published).order_by_views.take(6) %>
```

Comment:
@pranavrajs It appears we are fetching featured articles regardless of the selected locale. Should we add a locale filter as well?

![CleanShot 2023-10-26 at 13 08 01](https://github.com/chatwoot/chatwoot/assets/12408980/317e0425-31ab-4dd6-8a39-e3ee557b15db)

cc @scmmishra @iamsivin

Reply:
@iamsivin can you make this change?

Reply:
@muhsin-k @scmmishra Updated it here https://github.com/chatwoot/chatwoot/pull/8205/commits/22a9fd4d1f4b925ab4e3ecbbb7477c7f04dd08d8